lib/main_common.dart

```diff
@@ -87,9 +89,27 @@ Future<void> _firebaseInitializer(FlavorName flavorName) async {
   await Firebase.initializeApp(options: firebaseOption);
 }
 
+bool hasInterNetConnection = false;
+
 void _activateManagers() {
-  activateRemoteConfig();
+  final ConnectionStatus connectionStatus =
+      DependecyInjection.instance.get<ConnectionStatus>();
+
+  connectionStatus.initialize();
+  //Listen for connection change
+  connectionStatus.connectionChange
+      .asBroadcastStream()
+      .listen(connectionChanged);
+      debugPrint(connectionChanged.toString());
+
   if (!kIsWeb) {
     activateCrashReportIfEnabled();
   }
 }
+
+void connectionChanged(dynamic hasConnection) {
```

Comment:
Use specific type instead of dynamic

Reply:
The argument type 'void Function(bool)' can't be assigned to the parameter type 'void Function(dynamic)?'.
`StreamSubscription<dynamic> listen(
  void Function(dynamic)? onData, {
  Function? onError,
  void Function()? onDone,
  bool? cancelOnError,
})`